srcpkgs/pika-backup/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pika-backup'
+pkgname=pika-backup
+version=0.7.4
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config gettext itstool git"
+makedepends="libadwaita-devel gtk4-devel"
```

Comment:
add `rust-std`

Reply:
assumed build_helper would do that already, huh

Reply:
`build_style` does, not `build_helper`